.github/workflows/ci.yml

```diff
@@ -15,8 +15,6 @@ jobs:
       fail-fast: false
       matrix:
         node-version:
-          - 14
-          - 16
```

Comment:
with this PR, we're technically requiring Node 18 and above but I'll make that official in a separate PR.